test/soca/gw/CMakeLists.txt

```diff
@@ -32,8 +32,9 @@ set(jjob_list "JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP"
               "JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT"
               "JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT_VRFY"
               "JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN"
-              "JGDAS_GLOBAL_OCEAN_ANALYSIS_POST"
-              "JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY")
+              "JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT"
+              "JGDAS_GLOBAL_OCEAN_ANALYSIS_POST")
+#              "JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY")
```

Comment:
What's happening with VRFY?

Reply:
Waiting for your "post" PR to be merged in before we can run VRFY.

Reply:
The bug fix will be in @AndrewEichmann-NOAA 's PR @ShastriPaturi .